### PR TITLE
Support "supportsDebuggerProperties" from DAP

### PR DESCRIPTION
**Pull request by thaystg**
Implement support to Debugger Properties, then we can pass this information to BrowserDebugProxy and give the Wasm programmer a better debugging experience.

### CODE REVIEW

File: src/dap/api.d.ts [hunk 1]
```diff
@@ -1099,6 +1099,11 @@ export namespace Dap {
     setSourceMapSteppingRequest(
       params: SetSourceMapSteppingParams,
     ): Promise<SetSourceMapSteppingResult>;
+
+    /**
+     * Configures whether source map stepping is enabled.
```
Comment (radical) [inline src/dap/api.d.ts:1104]: `source map stepping` - comments needs to be updated

File: src/cdp/api.d.ts [hunk 2]
```diff
@@ -9639,6 +9640,31 @@ export namespace Cdp {
...
+   /**
+   * Types of the 'DotnetDebugger' domain.
+   */
+  export namespace DotnetDebugger {
+    /**
+     * Parameters of the 'DotnetDebugger.setDebuggerProperty' method.
+     */
+    export interface SetDebuggerPropertyParams {
+      /**
+       * Whether to enable to disable debugger property.
```
Comment (radical) [inline src/cdp/api.d.ts:9662]:
```suggestion
       * Whether to enable or disable debugger property.
```

File: src/adapter/debugAdapter.ts [hunk 3]
```diff
@@ -117,6 +117,11 @@ export class DebugAdapter implements IDisposable {
     this.dap.on('setExcludedCallers', params => this._onSetExcludedCallers(params));
     this.dap.on('saveDiagnosticLogs', ({ toFile }) => this._saveDiagnosticLogs(toFile));
     this.dap.on('setSourceMapStepping', params => this._setSourceMapStepping(params));
+    this.dap.on('setDebuggerProperty', params => this._setDebuggerProperty(params));
+  }
+
+  private _setDebuggerProperty(params: Dap.SetDebuggerPropertyParams) {
+    this._thread?.cdp().DotnetDebugger.setDebuggerProperty(params);
```
Comment (connor4312) [inline src/adapter/debugAdapter.ts:124]: Since the DAP request is generic, should this too have a more generic domain?

Comment (thaystg) [reply src/adapter/debugAdapter.ts:129]: Fixed.

**Review by connor4312 — APPROVED** [review]

**Pull request merged**